docs/source/bricks.rst

```diff
@@ -162,6 +162,36 @@ Examples:
     elements = partition_via_api(file=f, file_filename=filename, api_key="MY_API_KEY")
 
 
+``partition_multiple_via_api``
+------------------------------
+
+``partition_multiple_via_api`` is similar to ``partition_via_api``, but allows you to partition
+multiple documents in a single REST API call. The result has the type ``List[List[Element]]``.
```

Comment:
it would be worth mentioning what the return value looks like if one or more of the calls to the API fails.